docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,292 @@
+
+## Step 2. Set up a Scalar DL supported database 
+
+Scalar DL can run on various databases, which is an essential component for underlying storage. This section will help you to set up a Scalar DL supported database.
+
+### Requirements
+
+* You must have a Scalar DL supported managed database.
+
+Follow the [Set up a Scalar DL supported database](ScalarDLSupportedDatabase.md) document.
+
+## Step 3. Configure EKS
+
+Amazon EKS automatically manages the availability and scalability of the Kubernetes control plane nodes, so it will help you to achieve the high availability and scalability of Scalar DL. 
+This section will help you to configure a Kubernetes service for Scalar DL deployment.
+
+### Prerequisites
+Install the following tools on your host machine:
+* [awscli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install): In this tutorial, awscli is used to create a kubeconfig file to access the EKS cluster.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/): Kubernetes command-line tool to manage EKS cluster. Kubectl 1.19 or higher is required.
+
+### Requirements
+
+* You must have an EKS cluster with Kubernetes version 1.19 or above in order to use our most up-to-date configuration files.
+* Kubernetes node group must be labeled with key as `agentpool` and value as `scalardlpool` for ledger and envoy deployment.
+
+### Recommendations
+
+* Kubernetes node size should be `m5.large` for deploying `ledger` and `envoy` pods, because each node requires 2 vcpu and 8gb ram for deploying ledger and envoy pods.
+* Node groups should have at least 3 nodes for high availability in production.
+
+
+### Create a Kubernetes cluster	
+
+Kubernetes cluster is a primary component of EKS. Scalar DL requires a single EKS cluster for deploying ledger, envoy and monitor services. This section will help you to create an EKS cluster. 
+
+[Create an Amazon EKS cluster](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html) with the above requirements.
+
+> **Tip**
+>
+> If you are creating a Kubernetes cluster on a private subnet, you must add a new rule in the Kubernetes Security Group to enable HTTPS access to the public subnet from the Kubernetes cluster.
+
+### Create managed node groups
+
+This section will help you to create two managed node groups for Scalar DL deployment. 
+Create a managed node group for Ledger and Envoy deployment, and create another managed node group for deploying logs and metrics collection agents.
+
+[Create a managed node group](https://docs.aws.amazon.com/eks/latest/userguide/create-managed-node-group.html) with the above requirements and recommendations.
+
+## Step 4. Install Scalar DL
+
+Scalar DL is a blockchain-inspired tamper-evident distributed ledger software. 
```

Comment:
Why explaining Scalar DL here in a different way? It has already been explained at the top.

Reply:
Removed sentence since it's already described on top.